packages/nimble-components/src/banner/styles.ts

```diff
@@ -0,0 +1,197 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    BannerFail100DarkUi,
+    Black75,
+    Fail100LightUi,
+    Information100DarkUi,
+    Information100LightUi,
+    Warning100DarkUi,
+    Warning100LightUi,
+    White
+} from '@ni/nimble-tokens/dist/styledictionary/js/tokens';
+
+import {
+    actionRgbPartialColor,
+    applicationBackgroundColor,
+    bodyFont,
+    borderHoverColor,
+    buttonLabelFontColor,
+    controlHeight,
+    controlSlimHeight,
+    fillSelectedColor,
+    iconColor,
+    iconSize,
+    linkActiveFontColor,
+    linkDisabledFontColor,
+    linkFontColor,
+    standardPadding
+} from '../theme-provider/design-tokens';
+import { Theme } from '../theme-provider/types';
+import { hexToRgbaCssColor } from '../utilities/style/colors';
+import { themeBehavior } from '../utilities/style/theme';
+
+export const styles = css`
+    ${display('flex')}
+
+    :host {
+        width: 100%;
+        font: ${bodyFont};
+        font-size: 12.8px;
+        align-items: top;
+        overflow: hidden;
+        color: ${White};
+        ${iconColor.cssCustomProperty}: ${hexToRgbaCssColor(White, 0.6)};
+    }
+
+    :host(:not([open])) {
+        display: none;
+    }
+
+    .container {
+        display: flex;
+        width: 100%;
+    }
+
+    .icon {
+        width: 48px;
+        display: flex;
+        justify-content: center;
+        margin-top: 8px;
+        flex: 0 0 auto;
+    }
+
+    .text {
+        display: inline;
+        margin-top: 7px;
+        margin-bottom: 7px;
+    }
+
+    ::slotted([slot='title']) {
+        font-weight: bold;
+        padding-right: 8px;
+        white-space: nowrap;
+    }
+
+    .controls {
+        margin-left: auto;
+        display: flex;
+        justify-content: center;
+    }
+
+    slot[name='action'] {
+        display: flex;
+        align-content: center;
+    }
+
+    ::slotted(nimble-anchor[slot='action']) {
```

Comment:
The pattern I've seen throughout nimble is to write the selector like

```suggestion
    slot[name='action']::slotted(nimble-anchor) {
```

Assuming that works here, it would be nice for the selectors in this file to be consistent with other selectors in the repo.

Reply:
Fixed